src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -658,4 +671,19 @@ private EmailWrapper getEmptyEmailAddressedToEmail(String recipient) {
         return email;
     }
 
+    private String generateCoOwnersEmailsLine(String courseId) {
+        List<InstructorAttributes> coOwners = instructorsLogic.getCoOwnersList(courseId);
+        StringBuilder coOwnersEmailsList = new StringBuilder();
+        for (InstructorAttributes coOwner : coOwners) {
+            coOwnersEmailsList
+                    .append(coOwner.getName())
+                    .append('(')
+                    .append(coOwner.getEmail())
+                    .append("), ");
+        }
+        return coOwnersEmailsList.length() == 0
+                ? "(No contactable instructors found)"
```

Comment:
Since having no co-owners is an edge case and not part of the main success scenario, it may be preferable to do
```
if (coOwners.isEmpty()) {
    return "(No contactable instructors found)";
}
StringBuilder coOwnersEmailsList = ...
```